### PR TITLE
editcount got dropped in the move to update expressions

### PR DESCRIPTION
**Pull request by mcwhittemore**
@rclark looks like we missed this somehow. Does this look good?

### CODE REVIEW

**Comment by mcwhittemore** [comment]
This isn't ready yet, `calculateInfo` overwrites editcount

**Comment by mcwhittemore** [comment]
@rclark I accidentally added this to circle (which has now been removed) but the tests are passing. Am I good to merge?

File: lib/metadata.js [hunk 1]
```diff
@@ -137,8 +127,25 @@ function Metadata(dyno, dataset) {
...
+                        ':w': info.west, ':s': info.south, ':e': info.east, ':n': info.north,
+                        ':c': info.count, ':size': info.size, ':u': info.updated, ':ec': 0
+                    },
+                    UpdateExpression: 'set #w = :w, #s = :s, #e = :e, #n = :n, #c = :c, #size = :size, #u = if_not_exists(#u, :u), #ec = if_not_exists(#ec, :ec)',
+                    ConditionExpression: 'attribute_not_exists(#id)',
+                    ReturnValues: 'ALL_NEW'
+                };
+
+                dyno.updateItem(updateParams, function(err, update) {
+                    callback(err, prepare(update.Attributes));
```
Comment (rclark) [inline lib/metadata.js]: This would throw in the error case where `update` is undefined

Comment (mcwhittemore) [inline lib/metadata.js]: Oh, fixed.

**Comment by mcwhittemore** [comment]
@rclark nice catch. Did you want me to major bump this or is a patch cool?

**Comment by rclark** [comment]
Patch is fine. Thanks.

**Pull request merged**